packages/multichain/src/adapters/caip-permission-adapter-permittedChains.ts

```diff
@@ -57,7 +57,7 @@ export const addPermittedEthChainId = (
   caip25CaveatValue: Caip25CaveatValue,
   chainId: Hex,
 ): Caip25CaveatValue => {
-  const scopeString = `eip155:${parseInt(chainId, 16)}`;
+  const scopeString = `eip155:${hexToBigInt(chainId).toString(10)}`;
```

Comment:
I think this is one of the few times I've seen the chain ID converted to a bigint instead of a BN!